tests/bdd/features/volume_group/create/test_create.py

```diff
@@ -0,0 +1,258 @@
+"""Replica Anti-Affinity for Volume Group feature tests."""
+
+from typing import Dict
+from uuid import uuid4
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+    parsers,
+)
+
+import openapi.exceptions
+from common.apiclient import ApiClient
+from common.deployer import Deployer
+from openapi.model.create_pool_body import CreatePoolBody
+from openapi.model.create_volume_body import CreateVolumeBody
+from openapi.model.volume_policy import VolumePolicy
+
+NODE_NAME_1 = "io-engine-1"
+NODE_NAME_2 = "io-engine-2"
+NODE_NAME_3 = "io-engine-3"
+NON_VG_VOLUME_UUID = "5cd5378e-3f05-47f1-a830-a0f5873a1449"
+VG_VOLUME_UUID_1 = "c36f7453-ae65-43bb-8ee9-4b5a373a8ed4"
+VG_VOLUME_UUID_2 = "07805245-ec1c-43bd-a950-c42f4c3a9ac7"
+VG_VOLUME_UUID_3 = "6805e636-1986-4ab7-95d0-75597df1532b"
+VOLUME_SIZE = 10485761
+
+
+@scenario("create.feature", "creating volume group with 3 volumes with 1 replica each")
+def test_creating_volume_group_with_3_volumes_with_1_replica_each():
+    """creating volume group with 3 volumes with 1 replica each."""
+
+
+@scenario(
+    "create.feature",
+    "creating volume group with 3 volumes with 1 replica each with limited pools",
+)
+def test_creating_volume_group_with_3_volumes_with_1_replica_each_with_limited_pools():
+    """creating volume group with 3 volumes with 1 replica each with limited pools."""
+
+
+@scenario(
+    "create.feature", "creating volume group with 3 volumes with 3 replica on 3 pools"
+)
+def test_creating_volume_group_with_3_volumes_with_3_replica_on_3_pools():
+    """creating volume group with 3 volumes with 3 replica on 3 pools."""
+
+
+@scenario(
+    "create.feature", "creating volume group with 3 volumes with 3 replica on 5 pools"
+)
+def test_creating_volume_group_with_3_volumes_with_3_replica_on_5_pools():
+    """creating volume group with 3 volumes with 3 replica on 5 pools."""
+
+
+@scenario(
+    "create.feature", "creating volume group with 3 volumes with 3 replica on 9 pools"
+)
+def test_creating_volume_group_with_3_volumes_with_3_replica_on_9_pools():
+    """creating volume group with 3 volumes with 3 replica on 9 pools."""
+
+
+@given("2 pools, 1 on node A, 0 on node B, 1 on node C")
+def _():
```

Comment:
This should have a name, seems you're hitting the same issue as @dsharma-dc

Reply:
Any idea why this is happening? Its a pain to update all the method names manually, otherwise.